ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -1109,18 +1111,15 @@ def handle_post_reboot_test_reports(self):
             longest_downtime = (self.no_routing_stop - self.no_routing_start).total_seconds()
         else:
             longest_downtime = "N/A"
-        if self.no_routing_stop and self.reboot_start:
-            reboot_time = (self.no_routing_stop - self.reboot_start).total_seconds()
-        else:
-            reboot_time = "0:00:00"
-        if 'warm-reboot' in self.reboot_type and not self.kvm_test:
+        if self.total_disrupt_time:
             # Add total downtime (calculated in physical warmboot test using packet disruptions)
             total_downtime = self.total_disrupt_time
         else:
             total_downtime = "N/A"
         self.report["longest_downtime"] = longest_downtime
-        self.report["reboot_time"] = reboot_time
         self.report["total_downtime"] = total_downtime
+        self.report["lost_packets"] = self.total_disrupt_packets\
+            if self.total_disrupt_packets else "N/A"
```

Comment:
Just curious, why not 0? Does that mean something different than "N/A" here?

Reply:
"N/A" here indicates that `lost_packets` count is not valid for this run, as it is not calculated as part of the testcase. This would be the case in the fast-reboot test or KVM warm-reboot test).
But, if this was set to 0, it may falsely imply that there were no packet drops in this test.